diff --git a/nimble/host/src/ble_gap.c b/nimble/host/src/ble_gap.c
--- a/nimble/host/src/ble_gap.c
+++ b/nimble/host/src/ble_gap.c
@@ -3467,10 +3467,6 @@ ble_gap_event_connect_call(uint16_t conn_handle, int status)
     ble_gap_call_conn_event_cb(&event, handle);
 #endif
 
-    if (status == 0 ) {
-        ble_hs_hci_util_set_data_len(le16toh(conn_handle), BLE_HCI_SUGG_DEF_DATALEN_TX_OCTETS_MAX,
-                BLE_HCI_SUGG_DEF_DATALEN_TX_TIME_MAX);
-    }
 }
 
 void
